x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/trial_license_complete_tier/import_rules.ts

```diff
@@ -1376,6 +1378,65 @@ export default ({ getService }: FtrProviderContext): void => {
           });
         });
 
+        it('should resolve 100 or more exception references when importing into a clean slate', async () => {
+          // So importing a rule that references an exception list
```

Comment:
I was wondering why this comment was added, and then I saw that it was copied from an adjacent test 😭 .

I don't think the comments are needed; I would use either test structure or just more descriptive variable names to convey what these rules/exceptions represent:

```ts
// variable names:
const rulesWithExceptionLists = range(150)
...
const rulesWithExceptionListsPayload = combineArraysToNdJson();
```

```ts
// test structure:
describe('with a payload describing 150 rules, each with an exception list', () => {
  let saturatedRulePayload: string;
  beforeEach(() => {
    saturatedRulePayload = combineArraysToNdJson();
  });
  
  it('resolves the large payload when importing into a fresh environment');
});
```

Reply:
Oops yeah I glossed over the comments 😅 I removed those and fixed up the variable names a bit